ui/main/src/app/components/navbar/NavbarComponent.html

```
@@ -44,56 +47,60 @@
+                        *ngIf="menuElement.linkType === 'BOTH'"
+                        id="opfab-navbar-menu-icon-{{ menuElement.id }}"
+                        class="opfab-icon-external-link"
+                        (click)="clickOnMenu(menuElement, true)"
+                        (keydown.enter)="clickOnMenu(menuElement, true)"
+                        tabindex="0">
+                        <em class="fas fa-external-link-alt"></em>
+                    </a>
+                </li>
+                <li
+                    *ngIf="menuElement.dropdownMenu?.length > 0"
                     id="opfab-navbar-menu-{{ menuElement.id }}"
```

HTMLHint (1.5.0)

[error] The id value [ opfab-navbar-menu-{{ menuElement.id }} ] must be unique.

(id-unique)